Cargo.toml

```diff
@@ -6,7 +6,7 @@ description = "A system utility set"
 repository = "https://github.com/rust-vmm/vmm-sys-util"
 readme = "README.md"
 keywords = ["utils"]
-license = "Apache-2.0 OR BSD-3-Clause"
+license = "Apache-2.0 AND BSD-3-Clause"
```

Comment:
What about we also update the version to 0.1.1 and then publish the new version on crates.io?